src/main/java/nixy/exceptions/TaskNotExistException.java

```diff
@@ -0,0 +1,10 @@
+package nixy.exceptions;
+/**
+ * TaskNotExistException is a runtime exception
+ * thrown when an indicated task does not exist in the task list.
+ */
+public class TaskNotExistException extends NixyException {
```

Comment:
Perhaps this can be named `TaskDoesNotExistException` or `TaskNotFoundException` to ensure that the name is grammatically correct. This may help in readability.